### PR TITLE
Add malcolm detector name to ClusterProcessingModel

### PR DESCRIPTION
**Pull request by mpdickie**

### CODE REVIEW

File: org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/ClusterProcessingModel.java [hunk 1]
```diff
@@ -114,6 +127,7 @@ public int hashCode() {
 		final int prime = 31;
```
Comment (gerring) [inline org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/ClusterProcessingModel.java]: CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

**Comment by gerring** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ClusterProcessingModel.java#L141](https://github.com/eclipse/scanning/blob/46bc2912f2a058f903856fd2eee47c0879a156c6/org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/ClusterProcessingModel.java#L141): Refactor this method to reduce its Cognitive Complexity from 26 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by gerring** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 2 critical

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ClusterProcessingModel.java#L141](https://github.com/eclipse/scanning/blob/8b4a7fd75b901e6502db7d7ca163365f1e3e44dc/org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/models/ClusterProcessingModel.java#L141): Refactor this method to reduce its Cognitive Complexity from 26 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by mpdickie** [comment]
This pull request appears to be a duplicate.

**Pull request closed without merging**